### ISSUE TITLE
PM2 loses connection with Keymetrics on system reboot

### ISSUE THREAD

**Issue by pensierinmusica** [Unitech/pm2#2806]
Every time I restart the server Keymetrics shows it as offline. The only way to get it back there is to manually log in and run `pm2 link`. Then it instantly re-appears.

When I set up PM2 I did run `pm2 startup` and `pm2 save`, and it has correctly daemonized the PM2 managed processes. The only part that gets lost is the connection with Keymetrics.

Any idea how to fix it?

Thanks!

## I'm opening this issue because:

  - [X] pm2 isn't behaving like it should do.
  - [ ] pm2 is doing something I don't understand.
  - [ ] pm2 is crashing.
  - [ ] Other (_see at the end for feature requests_):

## What's going wrong?

On system reboot the server is unlinked from Keymetrics. The only way to get it back is to manually login and run `pm2 link`.

## General Informations:

 `pm2 -v` prints: 
 ``` 
2.4.3
 ```

`node -v` prints: 
``` 
v7.8.0
```

`uname -a` output for unix OR Windows version: 
```
Linux
 ```

**Comment by vmarchaud** [comment]
Can you paste here the content of `/etc/systemd/system/pm2.service` ? And can you confirm me that you have a `agent.json5` file inside your `~/.pm2` ?

**Comment by pensierinmusica** [comment]
Hi,

Yes I confirm that `agent.json5` is inside `~/.pm2`.

Regarding `/etc/systemd/system/pm2.service` instead there's no such file. The closest one I've found is `/etc/systemd/system/pm2-pi.service`, and its content is:

```sh
[Unit]
Description=PM2 process manager
Documentation=https://pm2.keymetrics.io/
After=network.target

[Service]
Type=forking
User=pi
LimitNOFILE=infinity
LimitNPROC=infinity
LimitCORE=infinity
Environment=PATH=/home/pi/.nvm/versions/node/v7.8.0/bin:/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
Environment=PM2_HOME=/home/pi/.pm2
PIDFile=/home/pi/.pm2/pm2.pid

ExecStart=/home/pi/.nvm/versions/node/v7.8.0/lib/node_modules/pm2/bin/pm2 resurrect
ExecReload=/home/pi/.nvm/versions/node/v7.8.0/lib/node_modules/pm2/bin/pm2 reload all
ExecStop=/home/pi/.nvm/versions/node/v7.8.0/lib/node_modules/pm2/bin/pm2 kill

[Install]
WantedBy=multi-user.target
```

**Comment by Unitech** [comment]
@pensierinmusica thanks for reporting
It has been fixed and PM2 2.4.4 has been released

**Comment by pensierinmusica** [comment]
Awesome, thanks!
